qldbdriver/session.go

```diff
@@ -75,7 +75,8 @@ func (session *session) wrapError(ctx context.Context, err error, transID string
 				abortSuccess:  true,
 				isISE:         false,
 			}
-		case http.StatusText(http.StatusInternalServerError), http.StatusText(http.StatusServiceUnavailable):
+		}
+		if awsErr.StatusCode() == http.StatusInternalServerError || awsErr.StatusCode() == http.StatusServiceUnavailable {
```

Comment:
Can you move this section to a `default:` section in the switch statement above? Just make the logic more obvious.

Reply:
We cant create default section because the default is L90-97

Reply:
sorry, my fault. you were right.